Core/CwCheat.cpp

```diff
@@ -340,7 +340,17 @@ std::vector<std::string> CWCheatEngine::GetCodesList() {
 		std::string line;
 		getline(list, line, '\n');
 
-		if (line.length() > 3 && (line.substr(0, 1) == "_" || line.substr(0, 2) == "//")) {
+		bool validCheatLine = false;
+		// This code is used by cheat menu which doesn't support empty names
+		if (line.length() >= 5 && line[0] == '_') {
+			for (int i = 4; i < line.length(); i++) {
```

Comment:
This is safe because we only care about `_C` lines, right?  I mean, `_G Nospaceshere` or `_S ULUS-12345` would otherwise be valid, right?

-[Unknown]

Reply:
It's safe because all lines that matter will need to have at least at least 1 non space character starting from 5 position.
We basically care about 2 things here:
- lines with name that has something which could be displayed, hence the loop breaks at first non space character and doesn't bother checking any syntax,
- lines which for sure does not have any valid cheat syntax.

 Which means too short, but also all lines that does not start from either:
````
_
//
#
````
all will be discarded whenever any change to the list is made:]. So if we paste a long novel inside cheat file, after that function runs, the cheat file will be cleaned ouf of all unrelated and uncommented text:].